### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by juliocoliveira** [uparlange/cemu-loader#1]
It starts with a blank screen.
![Screen](http://i.imgur.com/4yyVjuv.png)

**Comment by elcorsico** [comment]
This App requires NW.js:

Download lastest normal version of NW.js (https://nwjs.io/)

Unzip content in "C:\Program Files\nwjs"

Double click on the application (.nw file)

Select "nw.exe" from "C:\Program Files\nwjs" as default reader for "*.nw" files

**Comment by juliocoliveira** [comment]
I did it, if I didn't I would not be able to see this screen since the file is not an executable.

I found this debug.log inside the nwjs folder: 
```
[0825/104325.015:WARNING:chrome_main_delegate.cc(590)] final extension: .nw
[0825/104538.245:WARNING:chrome_main_delegate.cc(590)] final extension: .nw
[0825/104715.865:WARNING:chrome_main_delegate.cc(590)] final extension: .nw
[0825/104841.064:WARNING:chrome_main_delegate.cc(590)] final extension: .nw
```

**Comment by uparlange** [comment]
It was a bug with my last evolution.
Application does not load correctly if "cemu file" is not configured.
It is corrected in 1.3.0 version.
Have fun !

**Comment by juliocoliveira** [comment]
OK! Now is working!
